plugins/linux/l3plugin/descriptor/route.go

```diff
@@ -429,6 +440,17 @@ func (d *RouteDescriptor) Retrieve(correlate []adapter.RouteKVWithMetadata) ([]a
 	return values, nil
 }
 
+// resolves IP address family
+func (d *RouteDescriptor) isIPv6(address string) (isIPv6 bool) {
+	ipAddr := strings.Split(address, "/")
+	if len(ipAddr) < 1 {
+		d.log.Error("invalid IP address provided: %s", address)
+		return
+	}
+	ip := net.ParseIP(ipAddr[0])
+	return ip == nil || ip.To4() == nil
```

Comment:
I believe this condition is not correct and will return `true` even for any gibberish containing `/`.

If [`net.ParseIP`](https://pkg.go.dev/net#ParseIP) returns `nil` it means that given string is **not a valid textual representation of an IP address**.